src/Stratis.Features.FederatedPeg.Tests/CrossChainTransferStoreTests.cs

```diff
@@ -639,5 +639,48 @@ public async Task AttemptFederationInvalidWithdrawal()
                 return JsonConvert.DeserializeObject<Q>(result);
             }
         }
+
+
+        /// <summary>
+        /// Recording deposits when the wallet UTXOs are sufficient succeeds with deterministic transactions.
+        /// </summary>
+        [Fact]
+        public void StoringDepositsWhenTargetIsMultisigIsIgnored()
+        {
+            var dataFolder = new DataFolder(TestBase.CreateTestDir(this));
+
+            this.Init(dataFolder);
+            this.AddFunding();
+            this.AppendBlocks(WithdrawalTransactionBuilder.MinConfirmations);
+
+            MultiSigAddress multiSigAddress = this.wallet.MultiSigAddress;
+
+            using (ICrossChainTransferStore crossChainTransferStore = this.CreateStore())
+            {
+                crossChainTransferStore.Initialize();
+                crossChainTransferStore.Start();
+
+                TestBase.WaitLoopMessage(() => (this.ChainIndexer.Tip.Height == crossChainTransferStore.TipHashAndHeight.Height, $"ChainIndexer.Height:{this.ChainIndexer.Tip.Height} Store.TipHashHeight:{crossChainTransferStore.TipHashAndHeight.Height}"));
+                Assert.Equal(this.ChainIndexer.Tip.HashBlock, crossChainTransferStore.TipHashAndHeight.HashBlock);
+
+                // Forwarding money already in the multisig address to the muultisig address is ignored.
+                BitcoinAddress address1 = multiSigAddress.RedeemScript.Hash.GetAddress(this.network);
```

Comment:
Good, also do another transfer (another test is fine) where the opreturndata is a different multisig address.

Reply:
Done. Used the same test to save on test execution time.